helm/observability-bundle/values.yaml

```diff
@@ -49,6 +49,18 @@ apps:
                 # Add app label.
                 - targetLabel: app
                   replacement: kubernetes
+                metricRelabelings:
```

Comment:
Thanks!

Can we document this somewhere? 

I guess we have this ticket https://github.com/giantswarm/giantswarm/issues/24418

I think you can add this to the Prometheus knowledge area.

Reply:
I'm not sure what you would like to see documented?
* what is `metricRelabeling` and how it works?
* how to add relabelings in a servicemonitor?
* how to configure relabelings for the servicemonitors provided by prometheus-operator?

Maybe all of these :grinning: But that's a large subject!

Also, the ticket you linked is about prometheus data sent to grafana cloud. Which is mostly related to recording rules.
The current PR will change data ingested by local prometheis.

Reply:
2 things from me:
- do we know where apiserver_request_slo_duration_seconds_bucket is coming from? Because this is basically undocumented. 
- Should we remove apiserver_request_duration_seconds_bucket there as I would think this would be what Phoenix would use a an apiserver slo metric?

Reply:
> * Should we remove apiserver_request_duration_seconds_bucket there as I would think this would be what Phoenix would use a an apiserver slo metric?

The included drop for `apiserver_request_slo_duration_seconds_bucket` is already defined in the default serviceMonitor. It does not drop the whole metrics, but only some buckets. Meaning we lose precision in the histogram, but also reduce the amount of ingested data.